site/docs/4.3/getting-started/javascript.md

```diff
@@ -209,3 +209,45 @@ $('#yourTooltip').tooltip({
   }
 })
 {% endhighlight %}
+
+## Compatibility with IE 11
+
+Our v5 isn't designed to work with Internet Explorer 11, but you can add the following polyfills to make it work:
```

Comment:
Reading this paragraph, I'd expect it to list out the polyfills instead of showing a code snippet. Can we add what the polyfill.io script does and make a bulleted list before the code snippet?

Reply:
Initially the plan was to indeed list the polyfills. But we opted for the auto-detection.